nature/benchmarks/linear_mapper_benchmark.py

```diff
@@ -0,0 +1,181 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Linear Mapper Benchmarks."""
+from timeit import timeit
+import numpy as np
+import retworkx
+from qiskit_nature.mappers.second_quantization import LinearMapper
+from qiskit_nature.problems.second_quantization.lattice.models import IsingModel
+from qiskit_nature.problems.second_quantization.lattice import Lattice
+
+# pylint: disable=redefined-outer-name, invalid-name, attribute-defined-outside-init
+
+
+class LinearMapperBenchmarks:
+    """Linear Mapper Benchmarks."""
+
+    version = 1
+    timeout = 120.0
+    params = [50, 80, 100]
+    param_names = ["Number of nodes"]
+
+    def setup_cache(self):
+        """setup cache"""
+
+        np.random.seed(100)
+        second_q_ops_list = []
+        edge_arr = [
+            24,
+            67,
+            87,
+            79,
+            48,
+            10,
+            94,
+            52,
+            98,
+            53,
+            66,
+            98,
+            14,
+            34,
+            24,
+            15,
+            60,
+            58,
+            16,
+            9,
+            93,
```

Comment:
Yes, the numbers are random. However, since we were avoiding using numpy random, I added a static list of numbers to the program. These numbers basically serve as the weights of the edges in the graphs - I don't think their values would change the performance of the benchmark though.